### ISSUE TITLE
MapUpdateStressTest.testFixedCluster

### ISSUE THREAD

**Issue by hasancelik** [hazelcast/hazelcast#7105]
```
org.junit.runners.model.TestTimedOutException: test timed out after 300000 milliseconds
	at java.lang.Thread.sleep(Native Method)
	at com.hazelcast.client.stress.StressTestSupport.startAndWaitForTestCompletion(StressTestSupport.java:85)
	at com.hazelcast.client.stress.MapUpdateStressTest.test(MapUpdateStressTest.java:72)
	at com.hazelcast.client.stress.MapUpdateStressTest.testFixedCluster(MapUpdateStressTest.java:66)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-nightly/com.hazelcast$hazelcast-client/804/testReport/junit/com.hazelcast.client.stress/MapUpdateStressTest/testFixedCluster/

**Comment by hasancelik** [comment]
```
java.lang.RuntimeException: Could not join thread:Thread[com.hazelcast.client.stress.MapUpdateStressTest$StressThread1,5,main], thread is still alive
	at com.hazelcast.client.stress.StressTestSupport.joinAll(StressTestSupport.java:139)
	at com.hazelcast.client.stress.MapUpdateStressTest.test(MapUpdateStressTest.java:73)
	at com.hazelcast.client.stress.MapUpdateStressTest.testFixedCluster(MapUpdateStressTest.java:66)
```

https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.maintenance-nightly/com.hazelcast$hazelcast-client-new/728/testReport/junit/com.hazelcast.client.stress/MapUpdateStressTest/testFixedCluster/?auto_refresh=false

**Comment by cangencer** [comment]
Fixed in 3.5.x branch by #7328
